dap_api/app/app/tests/backend/test_ors_processor.py

```diff
@@ -57,6 +57,49 @@ def test_handle_ors_request(self, mocker: MockerFixture, db: Session, request_di
         # mock_requests
         assert out == res
 
+    @pytest.mark.parametrize(
+        "request_dict,options,header_authorization,out", [
+            (ORSDirections.parse_obj({
+                "portal_options": {"base_path": "disaster1"},
+                "coordinates": [
+                    [
+                        8.681495,
+                        49.41461
+                    ],
+                    [
+                        8.687872,
+                        49.420318
+                    ]
+                ]
+            }), PathOptions.parse_obj({
+                "portal_mode": "avoid_areas",
+                "ors_api": "directions",
+                "ors_profile": "driving-car",
+                "ors_response_type": "geojson"
+            }), "mock_api_key", "")
+        ]
+    )
+    def test_handle_ors_request_with_base_path(self, mocker: MockerFixture, db: Session, request_dict: ORSDirections | ORSIsochrones, options: PathOptions,
```

Comment:
If you only have a single array of parameters, you don't really need the parametrize.
It is for specifying multiple different inputs for a test.
In this case it would be cleaner to just define the variables directly in the test below (so you don't need to pass it to the test function as arguments).

Also the `out` which usually holds the output to assert against is not used in the test